adapters/mqtt-vertx-base/src/test/java/org/eclipse/hono/adapter/mqtt/AbstractVertxBasedMqttProtocolAdapterTest.java

```diff
@@ -1512,6 +1392,28 @@ private MqttEndpoint getMqttEndpointAuthenticated() {
         return getMqttEndpointAuthenticated("sensor1@DEFAULT_TENANT", "test");
     }
 
+    /**
+     * Creates a new adapter instance to be tested.
+     * <p>
+     * This method
+     * <ol>
+     * <li>creates a new {@code MqttServer} using {@link #getMqttServer(boolean)}</li>
+     * <li>assigns the result to property <em>server</em></li>
+     * <li>passes the server in to {@link #getAdapter(MqttServer)}</li>
+     * <li>assigns the result to property <em>adapter</em></li>
+     * </ol>
+     *
+     * @param configuration The configuration properties to use.
+     * @return The adapter instance.
+     */
+    private AbstractVertxBasedMqttProtocolAdapter<MqttProtocolAdapterProperties> givenAnAdapter(
+            final MqttProtocolAdapterProperties configuration) {
```

Comment:
The `configuration` parameter isn't used, so this looks somewhat confusing.

Also in the 2 other adapter test classes.

Reply:
Yes, I agree, that is not very nice. I have fixed it ...